src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -618,10 +642,27 @@ export async function containerDelete(projectInfo: ProjectInfo, script: string):
     processManager.killRunningProcesses(projectInfo.projectID, projectName);
 
     try {
+        let args: any[] = [projectInfo.location, LOCAL_WORKSPACE, projectID, "remove", containerName, undefined, undefined, undefined, undefined, undefined, undefined, deploymentRegistry];
+
+        if (projectInfo.projectType == "odo") {
```

Comment:
Why do we need to do this? Does Appsody has equivalent? We should try to avoid burning extension specific code into PFE. Can this be move to part of the extension code?